server/utils/utils.ts

```diff
@@ -55,4 +56,37 @@ const jsonToSimpleDateTime = (dt: string): SimpleDateTime | undefined => {
   return SimpleDateTime.fromSimpleDateAndTime(simpleDate, simpleTime)
 }
 
-export { convertToTitleCase, hasRole, simpleDateTimeToJson, jsonToSimpleDateTime }
+/**
+ * Converts an Address object to a comma-separated string
+ * @param address: SimpleDateTime
+ */
+const addressObjectToString = (address: Address): string => {
+  return Object.values(address).join(', ')
+}
+
+/**
+ * Converts a comma-separated string to an address object for display
+ * @param address
+ */
+const stringToAddressObject = (address: string): Address => {
+  if (!address) {
+    return undefined
+  }
+  const addressParts = address.split(', ')
```

Comment:
I guess string addresses are generated by your code so you can assume all the commas are there?